src/Config/Converter.php

```diff
@@ -26,6 +26,8 @@ class Converter implements ConverterInterface
             'delete_old_files' => ['type' => 'bool', 'default' => false],
             'archive_date_format' => ['type' => 'string', 'default' => 'dmYhis'],
             'directory_permissions' => ['type' => 'int', 'default' => 0777],
+            'archive_already_imported_files' => ['type' => 'bool', 'default' => false],
```

Comment:
So with that change you can request to not archive files? 
If so I would say it should be true by default, as import module is used to daily import routines, so files should be archived when processed.

In xsd I see that this property is not required. What value will we receive if this is not specified? Is it going to return always default value?

Reply:
This property is meant to be configurable only globally through config.php or env.php.
The usecase which it should solve is, that when module detects file which have the same content as already imported, it leave it in incoming folder (current archive_already_imported_files=false). With this option turned on, files marked as already imported are moved to failed directory.

Reply:
it will report that error once and file will be moved into error folder instead of staying in incoming

Reply:
I wonder what could be use case to have false here. Arent now importer files archives by default?

Reply:
IT seems to solve some annoying issues. Could you update readme with that option?